.github/checkgroup.yml

```diff
@@ -56,28 +60,27 @@ subprojects:
       # TODO: since this job cannot run on forks, it cannot be required or it will block all PL PRs from forks
       #- "test-on-tpus"
 
-  - id: "pytorch_lightning: Conda"
```

Comment:
Note: since this PR touches the job matrix and the configuration together, we will need an admin to force merge it